src/views/tickets/orders.php

```diff
@@ -105,45 +114,55 @@
 	<!-- Notices -->
 	<?php tribe_the_notices() ?>
 
-	<form method="post">
-
-		<?php $template->template( 'tickets/orders-rsvp' ); ?>
-
-		<?php
-		if ( ! class_exists( 'Tribe__Tickets_Plus__Commerce__PayPal__Meta' ) ) {
-			$template->template( 'tickets/orders-pp-tickets' );
-		}
-		?>
-
-		<?php
-		/**
-		 * Fires before the process tickets submission button is rendered
-		 */
-		do_action( 'tribe_tickets_orders_before_submit' );
-		?>
-
-		<?php if (
-			// Current user has RSVP (with or without meta) so needs to be able to edit status
-			$view->has_rsvp_attendees( $event_id, get_current_user_id() )
-			|| (
-				// Current user has tickets with meta so needs to be able to edit meta
-				$view->has_ticket_attendees( $event_id, get_current_user_id() )
-				&& $tribe_my_tickets_have_meta
-			)
-		) : ?>
-			<div class="tribe-submit-tickets-form">
-				<button
-					type="submit"
-					name="process-tickets"
-					value="1"
-					class="button alt"
-				>
-					<?php echo sprintf( esc_html__( 'Update %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, get_current_user_id() ) ); ?>
-				</button>
-			</div>
-		<?php endif;
-		// unset our global since we don't need it any more
-		unset( $tribe_my_tickets_have_meta );
-		?>
-	</form>
+	<div
+		class="tribe-tickets__tickets-page-wrapper"
+		data-post-id="<?php echo esc_attr( $event_id ); ?>"
+		data-provider="<?php echo esc_attr( $provider ); ?>"
+		data-resending-allowed="<?php echo esc_attr( $allow_resending_email ); ?>"
+	>
+
+		<form method="post" autocomplete="off">
+
+			<?php $template->template( 'tickets/orders-rsvp' ); ?>
+
+			<?php
+			if ( ! class_exists( 'Tribe__Tickets_Plus__Commerce__PayPal__Meta' ) ) {
+				$template->template( 'tickets/orders-pp-tickets' );
+			}
+			?>
+
+			<?php
+			/**
+			 * Fires before the process tickets submission button is rendered
+			 */
+			do_action( 'tribe_tickets_orders_before_submit' );
+			?>
+
+			<?php if (
+				// Current user has RSVP (with or without meta) so needs to be able to edit status
+				$view->has_rsvp_attendees( $event_id, get_current_user_id() )
+				|| (
+					// Current user has tickets with meta so needs to be able to edit meta
+					$view->has_ticket_attendees( $event_id, get_current_user_id() )
+					&& $tribe_my_tickets_have_meta
+				)
+			) : ?>
+				<div class="tribe-submit-tickets-form">
+					<button
+						type="submit"
+						name="process-tickets"
+						value="1"
+						class="button alt"
+					>
+						<?php echo sprintf( esc_html__( 'Update %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, get_current_user_id() ) ); ?>
+					</button>
+				</div>
+			<?php endif;
+			// unset our global since we don't need it any more
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a0243db49e8b54799bd0d7f5ff875b54</sub>